shell/platform/windows/cursor_handler.cc

```diff
@@ -11,9 +11,36 @@
 static constexpr char kChannelName[] = "flutter/mousecursor";
 
 static constexpr char kActivateSystemCursorMethod[] = "activateSystemCursor";
-
 static constexpr char kKindKey[] = "kind";
 
+// This method allows creating a custom cursor with rawBGRA buffer, returns a
+// string to identify the cursor.
+static constexpr char kCreateCustomCursorMethod[] =
+    "createCustomCursor/windows";
+// A string, the custom cursor's name.
+static constexpr char kCustomCursorNameKey[] = "name";
+// A list of bytes, the custom cursor's rawBGRA buffer.
+static constexpr char kCustomCursorBufferKey[] = "buffer";
+// A double, the x coordinate of the custom cursor's hotspot, starting from
+// left.
+static constexpr char kCustomCursorHotXKey[] = "hotX";
+// A double, the y coordinate of the custom cursor's hotspot, starting from top.
+static constexpr char kCustomCursorHotYKey[] = "hotY";
+// An int value for the width of the custom cursor.
+static constexpr char kCustomCursorWidthKey[] = "width";
+// An int value for the height of the custom cursor.
+static constexpr char kCustomCursorHeightKey[] = "height";
+
+// This method allows setting a custom cursor with a unique int64_t key of the
+// custom cursor.
+static constexpr char kSetCustomCursorMethod[] = "setCustomCursor/windows";
+// A string value for the key of the custom cursor.
+static constexpr char kCustomCursorKey[] = "key";
```

Comment:
Why is this `key`? Shouldn't it just use `kCustomCursorNameKey`?

Reply:
my apology that I forget to uniform this key. Now it's has been resolved.